### PR TITLE
python-aiohttp: update to version 3.7.4 (security fix)

### PR DESCRIPTION
**Pull request by ja-pa**
Maintainer: @BKPepe 
Compile tested: Turris Omnia (TOS6), OpenWrt master
Run tested: Turris Omnia (TOS6), OpenWrt master

Description:
This PR  updates python-aiohttp to version 3.7.4. It fixes [CVE-2021-21330](https://nvd.nist.gov/vuln/detail/CVE-2021-21330) and adds new package python-typing-extensions which is new aiohttp dependency.

[Changelog](https://github.com/aio-libs/aiohttp/blob/master/CHANGES.rst#374-2021-02-25)

Run tested with simple aiohttp example

### CODE REVIEW

File: lang/python/python-typing-extensions/Makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-typing-extensions
```
Comment (BKPepe) [inline lang/python/python-typing-extensions/Makefile:10]:
```suggestion
PKG_NAME:=python-typing-extensions
```

It should match the folder name

**Review by BKPepe — APPROVED** [review]
Otherwise LGTM, thank you!

File: lang/python/python-typing-extensions/Makefile (hunk 1 above)
Comment (ja-pa) [reply lang/python/python-typing-extensions/Makefile]: Thanks, fixed

File: lang/python/python-typing-extensions/Makefile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-typing-extensions
+PKG_VERSION:=3.7.4.3
+PKG_RELEASE:=1
+
+PYPI_NAME:=typing_extensions
```
Comment (jefferyto) [inline lang/python/python-typing-extensions/Makefile:14]:
```suggestion
PYPI_NAME:=typing-extensions
PYPI_SOURCE_NAME:=typing_extensions
```

**Review by jefferyto — COMMENTED** [review]
I know this is super pedantic but I'd really like `PYPI_NAME` to match the library name in [PyPI](https://pypi.org/project/typing-extensions/).

File: lang/python/python-typing-extensions/Makefile (hunk 2 above)
Comment (ja-pa) [reply lang/python/python-typing-extensions/Makefile]: Fixed. Thanks for the suggestion. Next time I will use PYPI_SOURCE_NAME in a similar case.

**Pull request merged**